Documentation/gettingstarted/bgp.rst

```diff
@@ -0,0 +1,168 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _bgp:
+
+**********
+BGP (beta)
+**********
+
+BGP provides a way to advertise routes using traditional networking protocols
+to allow Cilium-managed services to be accessible outside the cluster.
+
+This document explains how to configure Cilium's native support for announcing
+``LoadBalancer`` IPs of ``Services`` via BGP. It leverages `MetalLB's
+<https://metallb.universe.tf/>`_ simple and effective implementation of IP
+allocation and the minimal BGP protocol support to do this. The configuration
+for Cilium is the same as MetalLB's configuration.
+
+Specifically, if a ``Service`` of type ``LoadBalancer`` is created, Cilium will
+allocate an IP for it from a specified pool. Once the IP is allocated, the
+Agents will announce via BGP depending on the ``Service``'s
+``ExternalTrafficPolicy``. See MetalLB's `documentation
+<https://metallb.universe.tf/usage/#bgp>`_ on this specific topic.
+
+.. include:: ../beta.rst
+
+Deploy Cilium
+===============
```

Comment:
```suggestion
=============
```

Reply:
Thanks, fixed. Even with the align column in Vim, I still missed it :sweat_smile:

Reply:
Just don't tell that to the Goland aficionados :)